### DISCUSSION TITLE
Can't get (intruded) event to trigger

### DISCUSSION THREAD

**Discussion by dhamekasi** [ts1/BLEUnlock#114] [Q&A]
Hey! great app, thanks for sharing.

Using an apple watch as my bluetooth device.

My script works fine for away,lost and unlocked, but the app never seems to trigger the intruded event, regardless of whether I unlock via password, native apple watch unlock or touch id.

unlocked) works fine if I use BLEUnlock's unlock function.

I have made sure the preference "require password after lock" is set to immediately.

any ideas?

Mac OS Sonoma 14.2.1

**Comment by ts1** [comment]
I've also noticed the same issue when using my Apple Watch for unlocking. With today's release, the 'intruded' event now triggers correctly during 'Wake without Unlocking' scenarios.
